scripts/private-archive-preprocess/esa_cci_gravimetric_mass_balance_dtu/gmb_dtu_nc_to_gpkg.py

```diff
@@ -7,14 +7,15 @@
 
 DATASET_ARCHIVE_LOCATION = '/share/appdata/qgreenland-private-archive/esa_cci_gravimetric_mass_balance_dtu/'
 DATA_PATH = os.path.join(DATASET_ARCHIVE_LOCATION, 'greenland_gravimetric_mass_balance_rl06_dtuspace_v2_0-170820/CCI_GMB_GIS.nc')
-OUTPUT_SHAPEFILE_PATH = os.path.join(DATASET_ARCHIVE_LOCATION, 'QGREENLAND_SHAPEFILES')
+OUTPUT_GPKG_PATH = os.path.join(DATASET_ARCHIVE_LOCATION, 'QGREENLAND_GEOPACKAGES')
```

Comment:
TODO: We probably use this path in multiple places too, let's also extract this.

Reply:
Nevermind, this isn't a task like I thought it was